### PR TITLE
docs: Hyperlink to W3Schools to learn HTML

### PR DESCRIPTION
**Pull request by ghost**
This link will help in learning the basic knowledge on HTML.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/docs.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Tutorials and guides include downloadable example to accelerate your projects.
 ## Assumptions
 
 
-These docs assume that you are already familiar with HTML, CSS, [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript "Learn JavaScript"), 
+These docs assume that you are already familiar with [HTML](https://www.w3schools.com/html/ "Learn HTML"), CSS, [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript "Learn JavaScript"),
```
Comment (gkalpak) [inline aio/content/marketing/docs.md]: Let's add links to MDN (if nothing else, to be consistent) for both HTML (https://developer.mozilla.org/docs/Learn/HTML/Introduction_to_HTML) and CSS (https://developer.mozilla.org/docs/Learn/CSS/First_steps).

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @dishanfernando 👍
I left a suggestions. LMK what you think.

File: aio/content/marketing/docs.md (hunk 1 above)
Comment (dishanfernando) [reply aio/content/marketing/docs.md]: Sure. That would be good.

Comment (IgorMinar) [reply aio/content/marketing/docs.md]: +1 for MDN, w3schools' content is problematic and we should not be pointing to it.

Comment (dishanfernando) [reply aio/content/marketing/docs.md]: Thanks for the info

**Review by bofeiw — APPROVED** [review]

**Comment by gkalpak** [comment]
Superceded by #33844.

**Pull request closed without merging**